C/CoinUtils/build_tarballs.jl

```diff
@@ -0,0 +1,82 @@
+                  "d4effff4452e73356eed9f889efd9c44fe9cd68bd37b608a5ebb2c58bd45ef81"),
+    # DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/CoinUtils-releases-2.11.4
+
+# Remove wrong libtool files
+rm -f /opt/${target}/${target}/lib*/*.la
+
+# if [[ "${target}" == *-musl* ]]; then
+#     # This is to fix the following error:
+#     #    node_heap.cpp:11:22: fatal error: execinfo.h: No such file or directory
+#     #     #include <execinfo.h>
+#     # `execinfo.h` is GlibC-specific, not Linux-specific
+#     atomic_patch -p1 "${WORKSPACE}/srcdir/patches/glibc_specific.patch"
+# fi
+
+OPENBLAS=(-lopenblas)
+CPPFLAGS=()
+if [[ ${nbits} == 64 ]] && [[ ${target} != aarch64* ]]; then
+    OPENBLAS=(-lopenblas64_)
+  syms=(caxpy cbdsqr ccopy cdotc cdotu cgbmv cgbtrf cgemm cgemv cgerc cgeru cgetrf cgetrs chbmv chemm chemv cher cher2 cher2k cherk chetrd clacgv clacpy cladiv clanhs clarfg clartg claset clasr classq claswp cpbtrf cpotrf cpttrf crot csbmv cscal csscal cswap csymm csyr2k csyrk ctbtrs ctrmm ctrmv ctrsm ctrsv ctrtrs dasum daxpy dsbmv dbdsqr dcopy ddot dgbmv dgbtrf dgemm dgemv dger dgetrf dgetrs dhbmv disnan dlabad dlacpy dlae2 dlaebz dlaed4 dlaev2 dlagtf dlagts dlahqr dlamc3 dlamch dlange dlanst dlanv2 dlapy2 dlapy3 dlaqr0 dlaqr1 dlaqr3 dlaqr4 dlarfg dlarfx dlarnv dlarra dlarrb dlarrc dlarrd dlarrk dlarrv dlartg dlaruv dlascl dlaset dlasq2 dlasr dlasrt dlassq dlaswp dlasy2 dnrm2 dpbtrf dpotrf dpttrf drot dscal dstedc dsteqr dsterf dswap dsymm dsymv dsyr dsyr2 dsyr2k dsyrk dsytrd dtbtrs dtrmm dtrmv dtrsm dtrsv dtrtrs dzasum dznrm2 dzsum1 icamax icmax1 idamax ieeeck ilaenv isamax izamax izmax1 lsame lsamen sasum saxpy sbdsqr scasum scnrm2 scopy scsum1 sdot sgbmv sgbtrf sgemm sgemv sger sgetrf sgetrs shbmv sisnan slabad slacpy slae2 slaebz slaed4 slaev2 slagtf slagts slahqr slamc3 slamch slange slanst slanv2 slapy2 slapy3 slaqr0 slaqr1 slaqr3 slaqr4 slarfg slarfx slarnv slarra slarrb slarrc slarrd slarrk slarrv slartg slaruv slascl slaset slasq2 slasr slasrt slassq slaswp slasy2 snrm2 spbtrf spotrf spttrf srot ssbmv sscal sstedc ssteqr ssterf sswap ssymm ssymv ssyr ssyr2 ssyr2k ssyrk ssytrd stbtrs strmm strmv strsm strsv strtrs xerbla zaxpy zbdsqr zcopy zdotc zdotu zdscal zgbmv zgbtrf zgemm zgemv zgerc zgeru zgetrf zgetrs zhbmv zhemm zhemv zher zher2 zher2k zherk zhetrd zlacgv zlacpy zladiv zlanhs zlarfg zlartg zlaset zlasr zlassq zlaswp zpbtrf zpotrf zpttrf zrot zdbmv zscal zswap zsymm zsyr2k zsyrk ztbtrs ztrmm ztrmv ztrsm ztrsv ztrtrs)
+  for sym in ${syms[@]}
+  do
+    CPPLAGS+=("-D${sym}=${sym}_64")
+  done
+fi
+
+configure_args=()
+if [[ ${target} == *darwin* || ${target} == arm-* || ${target} == *freebsd* ]]; then
+  configure_args+=(--host=${target})
+fi
+
+CPPFLAGS="${CPPFLAGS[@]}" ./configure --prefix=${prefix} --with-blas-lib="${OPENBLAS[@]}" ${configure_args[@]}
+make -j${nproc}
```

Comment:
Is there a reason to do `--host` only selectively? In most packages I am able to use it everywhere. Perhaps good to add the autoconf update script that @giordano mentioned as well.

Reply:
> Is there a reason to do `--host` only selectively?

Yes, this feels very wrong.  `--host` is needed to tell what's our target platform.  If you need to exclude it, there is something not working as it should.

Reply:
It's not that I have to excluded. It's only necessary on some platforms.

Reply:
> It's only necessary on some platforms.

It doesn't hurt to spell out precisely what the target platform is.  What you're doing adds unnecessary complexity, and I think you're missing powerpc, too.  Just always pass `--host` as usual.

Reply:
Ok